README.rst

```diff
@@ -254,6 +254,66 @@ To configure any Fedora 20 bare metal machine (e.g. on OnMetal as above)::
 
 Where ``${PASSWORD}`` is the password in ``slaves.fedora-vagrant.passwords`` from the ``config.yml`` or ``staging.yml`` file used to deploy the BuildBot master on hostname or IP address ``${MASTER}``.
 
+PistonCloud
+-----------
+
+The following builders need to run on Centos-7 on PistonCloud:
+
+* flocker-openstack-pistoncloud
+
+To create this machine you'll need to access various machines within pistoncloud via an "SSH jump host".
+
+The machines are referred to here as:
+ * pistoncloud-jumphost: The server through which you will connect to servers inside the pistoncloud network.
+ * pistoncloud-novahost: The server which has ``novaclient`` and associated command line tools installed.
+ * pistoncloud-buildslave: The server which we have created to run the ClusterHQ buildslave.
+
+The easiest way to access these machines is to create an SSH config file containing the hostnames and usernames that you will use to access the machines.
```

Comment:
If the latter then it would be really nice to create this file automatically, or have a `.sample` in the repo like with `config.yml`.